shared/src/test/kotlin/com/egm/stellio/shared/util/JsonLdUtilsTests.kt

```diff
@@ -95,4 +98,42 @@ class JsonLdUtilsTests {
         Assertions.assertTrue(resultMap.containsKey("location"))
         Assertions.assertFalse(resultMap.containsKey("isParked"))
     }
+
+    @Test
+    fun `it should throw an InvalidRequest exception if the JSON-LD fragment is not a valid JSON document`() {
+        val rawEntity =
+            """
+            {
+                "id": "urn:ngsi-ld:Device:01234",,
+                "type": "Device"
+            }
+            """.trimIndent()
+
+        val exception = assertThrows<InvalidRequestException> {
+            parseAndExpandJsonLdFragment(rawEntity)
+        }
+        Assertions.assertNotNull(exception.message)
```

Comment:
why aren't you testing the content of the message (as in the next test)?

Reply:
Added